web/styles/inbox.css

```diff
@@ -589,7 +599,9 @@
 }
 
 .recent-view-filter-dropdown-list-container .dropdown-list-item-description,
-.inbox-filter-dropdown-list-container .dropdown-list-item-description {
+.inbox-filter-dropdown-list-container .dropdown-list-item-description,
+.channel_privacy-dropdown-list-container .dropdown-list-item-description,
+.new_channel_privacy-dropdown-list-container .dropdown-list-item-description {
```

Comment:
These feel overdue for having an option to the dropdown widget for some named option for dropdown widgets with this styling, since we're using it in 4 places now.

Maybe start a thread in #frontend to discuss with Aman what to call it?